### PR TITLE
Catalinazoldi/capitolul8

### PR DESCRIPTION
**Pull request by zoldicatalina**

### CODE REVIEW

File: src/test/java/catalinazoldi/tests/Tema9/Tema7/CURAHealthcare.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
+package catalinazoldi.tests.Tema9.Tema7;
+
+import catalinazoldi.tests.Tema7.BaseTest;
```
Comment (95teo) [inline src/test/java/catalinazoldi/tests/Tema9/Tema7/CURAHealthcare.java:3]: Replace this import with the one from Tema9 package -> `import catalinazoldi.tests.Tema9.Tema7.BaseTest;`

File: src/test/java/catalinazoldi/tests/Tema9/Tema8/tests/BasicAjaxTest.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package catalinazoldi.tests.Tema9.Tema8.tests;
+
+
+import catalinazoldi.tests.Tema8.pageObjects.BasicAjaxPage;
```
Comment (95teo) [inline src/test/java/catalinazoldi/tests/Tema9/Tema8/tests/BasicAjaxTest.java:4]: Replace this import with the one from Tema9 package -> `import catalinazoldi.tests.Tema9.Tema8.pageObjects.BasicAjaxPage;`

File: src/test/java/catalinazoldi/tests/Tema9/Tema8/tests/DynamicButtonsDisabledTest.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package catalinazoldi.tests.Tema9.Tema8.tests;
+
+
+import catalinazoldi.tests.Tema8.pageObjects.DynamicButtonsDisabledPage;
```
Comment (95teo) [inline src/test/java/catalinazoldi/tests/Tema9/Tema8/tests/DynamicButtonsDisabledTest.java:4]: Replace this import with the one from Tema9 package -> `import catalinazoldi.tests.Tema9.Tema8.pageObjects.DynamicButtonsDisabledPage;`

File: src/test/java/catalinazoldi/tests/Tema9/Tema8/tests/DynamicButtonsSimpleTest.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
+package catalinazoldi.tests.Tema9.Tema8.tests;
+
+import catalinazoldi.tests.Tema8.pageObjects.DynamicButtonsSimplePage;
```
Comment (95teo) [inline src/test/java/catalinazoldi/tests/Tema9/Tema8/tests/DynamicButtonsSimpleTest.java:3]: Replace this import with the one from Tema9 package -> `import catalinazoldi.tests.Tema9.Tema8.pageObjects.DynamicButtonsSimplePage;`

File: src/test/java/catalinazoldi/tests/Tema9/Tema8/tests/RedirectTest.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
+package catalinazoldi.tests.Tema9.Tema8.tests;
+
+import ancaMarian.tests.tema8.pageObjects.JavaScriptRedirectPage;
```
Comment (95teo) [inline src/test/java/catalinazoldi/tests/Tema9/Tema8/tests/RedirectTest.java:3]: Replace this import with the one from Tema9 package -> `import catalinazoldi.tests.Tema9.Tema8.pageObjects.RedirectPage;`

File: src/test/java/catalinazoldi/tests/Tema9/Tema8/tests/RefreshTest.java [hunk 6]
```diff
@@ -0,0 +1,24 @@
+package catalinazoldi.tests.Tema9.Tema8.tests;
+
+import ancaMarian.tests.tema8.pageObjects.RefreshPage;
```
Comment (95teo) [inline src/test/java/catalinazoldi/tests/Tema9/Tema8/tests/RefreshTest.java:3]: Replace this import with the one from Tema9 package -> `import catalinazoldi.tests.Tema9.Tema8.pageObjects.RefreshPage;`

**Review by 95teo — CHANGES_REQUESTED** [review]

**Review by 95teo — APPROVED** [review]

**Review by ioanaandroflorea — APPROVED** [review]

**Pull request merged**